packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -388,6 +388,10 @@ export function getCurrentEventPriority(): EventPriority {
   return getEventPriority(currentEvent.type);
 }
 
+export function getIsCurrentEventPopState(): boolean {
```

Comment:
Let's call this something more generic since it's an abstraction inside Fiber. Like `shouldAttemptEagerTransition()`.